queries/highlights.scm

```diff
@@ -240,14 +240,17 @@ keyword: (identifier) @keyword
 [
   (block_comment)
   (comment)
-] @comment @spell
+] @spell
```

Comment:
I'm not sure what `@spell` supposed to represent. I'm keeping it here just in case it's important.